lgc/patch/PatchResourceCollect.cpp

```diff
@@ -2911,8 +2962,12 @@ void PatchResourceCollect::scalarizeGenericOutput(CallInst *call) {
 // @param call : Call to process
 // @param shaderStage : Shader stage
 void InOutLocationInfoMapManager::createMap(const std::vector<CallInst *> &calls, ShaderStage shaderStage) {
+  // LDS load/store copes with dword. So for 8-bit/16-bit data type, we will extend them to 32-bit for a LocationSpan
+  const bool requireDword =
+      shaderStage == ShaderStageTessControl || shaderStage == ShaderStageGeometry ||
+      (shaderStage == ShaderStageFragment && m_pipelineState->hasShaderStage(ShaderStageGeometry));
```

Comment:
It seems m_pipelineState is only used in this function, how about add parameter bool requireDword, instead of add a class member m_pipelineState